var/spack/repos/builtin/packages/py-fenics-fiat/package.py

```diff
@@ -23,10 +23,6 @@ class PyFenicsFiat(PythonPackage):
 
     version("master", branch="master")
     version('2019.1.0',       sha256='2a6d175a825ed725843918ef28846edbcf710a879c2fe8caaeda77b1ce9b9a1c')
```

Comment:
Why are these versions deleted?

Reply:
The intend was to support the 2019.1.0 version of fenics only, hence to delete the old versions within the dependent python packages (ffc, ufl, fiat, dijitso). However I agree it might be easier to keep the ufl, fiat and dijitso package files as they are and to refer to the required version in the fenics2019 and ffc package.

Reply:
@emai-imcs Since `spack` is about providing multipler versions and allow each user to pick its favorite at install time, it's actually intended to keep the other versions around.

Reply:
See #20767 and #15896 for the deprecation model I would like to move towards